src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
+const populationStrings = [];
+let total = 0;
+
+for (let i = 0; i < population.length; i++) {
+  const string = population[i].innerText.replace(new RegExp(',', 'g'), '');
+
+  populationStrings.push(Number(string));
+  total += populationStrings[i];
```

Comment:
I think you dont need ```populationStrings``` array ( Why you call it strings if you are pushing numbers?). You could simply:
```suggestion
  total += Number(string);
```